app/controllers/newsletter_controller.rb

```diff
@@ -0,0 +1,14 @@
+class NewsletterController < ApplicationController
+
+  def subscribe
+    email = params[:email]
+    list = params[:list]
+    begin
+      gibbon = Gibbon::Request.new(api_key: ENV["MAILCHIMP_API"])
+      gibbon.lists(ENV[list]).members.create(body: {email_address: email, status: "subscribed"})
+      render :json => { }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside empty hash literal braces detected.